pkgs/development/python-modules/crc32c/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  version = "2.0";
+  pname   = "crc32c";
+
+  src = fetchFromGitHub {
+    owner = "ICRAR";
+    repo  = pname;
+    rev   = "v${version}";
+    sha256 = "15x1sj23n50qdjmi8mjq5wgf5jfn1yv78vjc59wplvl0s50w2dnk";
+  };
+
+  meta = {
+    inherit (src.meta) homepage;
```

although this follows the DRY principle, at least personally I would prefer a raw url in which i can copy and quickly view which project this is.
```suggestion
    homepage = "https://github.com/ICRAR/crc32c";
```

---

Done.